fe/src/components/Drawerbar/DrawerbarContent.tsx

```diff
@@ -0,0 +1,17 @@
+import { Flex } from '@chakra-ui/react';
+// eslint-disable-next-line
+import { IoHome } from 'react-icons/io5';
+import React from 'react';
+import SidebarItemLink from '../custom/SidebarItemLink';
+import SidebarItem from '../custom/SidebarItem';
+
+export const DrawerbarContent: React.FC = () => (
+  <Flex direction="column">
+    <SidebarItem>
+      <SidebarItemLink icon={<IoHome size={22} />} title="Home" where="/" />
```

Comment:
Please make the Navbar and the Drawbar use the same links so it wouldn't have to be duplicated
https://github.com/TPT-Loane/TPT-Loane/blob/main/fe/src/components/Header/Header.tsx#L14

Reply:
You mean the same component? SidebarItemLink

Reply:
I guess you could use the same component if you wanted to, but I just want the links to come from one place so they won't be duplicated

Reply:
I don't get what you mean
They look different so they are not the same components

Reply:
You can keep them as separate components if you want I just want one source of truth for the links. Currently both your code has links hard-coded and the header code here https://github.com/TPT-Loane/TPT-Loane/blob/main/fe/src/components/Header/Header.tsx#L14-L23

It will take very little time until somebody forgets to update on or the other component and we'll end up with broken navigation.

Reply:
fixed it